addons/main/stringtable.xml

```diff
@@ -23,492 +23,569 @@
             <German>Stalker Anomalien</German>
             <Russian>Аномалии</Russian>
             <Polish>Stalker Anomalies</Polish>
+			<French>Stalker anomalies</French>
```

Comment:
Thank you for your contribution, we are using spaces instead of tabs. Could you convert you changes to use spaces instead?